### DISCUSSION TITLE
Cannot log in to my second account

### DISCUSSION THREAD

**Discussion by silentlight** [vercel/community#4011] [Account Issue]
### Question

I am logged in to my Vercel using Github. I should be owner of Inspotle team but I don't see it in teams dropdown. But when I go to suggested team I see Inspotle but it says Request Access. When I request access, receive email and click on it. It says 404 wrong user. What is going on? I am getting billed for project I don't have access to.

**Comment by amyegan** [comment]
Hey @silentlight. Sorry you're having trouble with your account. We can't safely discuss account information in public here. Please email [registration@vercel.com](mailto:registration@vercel.com) to open a ticket with the team that can help with login issues. 🙏
